deps/jerry/tests/jerry/array-prototype-includes.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+  console.log([].includes.call(arguments, 'a')); // true
+  console.log([].includes.call(arguments, 'd')); // false
+})('a','b','c');
+
+// 数组长度是3
```

Comment:
Would you please use english?

Reply:
sorry, forget to remove hints fro myself, i will remove it later